CHANGELOG.md

```diff
@@ -4,8 +4,22 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 ## [Unreleased]
- 
-## [3.2.0] - 2018-11-XX
+
+### Added
+- Metrics for BlobStore
+- New Guice product using Cassandra RabbitMQ ElasticSearch, OpenStack Swift and optional LDAP dependency (experiemental)
```

Comment:
you didn't fixed the type on experimental

Reply:
True, sorry.